bin/newman.js

```diff
@@ -85,6 +85,42 @@ program
         });
     });
 
+// The `request` command allows you to run single request with the provided options.
+program
+    .command('request <url>')
+    .description('Run single request with provided options')
+    .usage('<url> [options]')
+    .option('-X, --request <method>',
```

Comment:
Just a caveat to be aware of while handling this option, `-X` is given preference over the native `--GET` option i.e. when both the options are provided, `--GET` will be ignored and the method in `-X` will be used.
```sh
# would resolve in PUT
$ curl --GET -X PUT https://google.com 

# would resolve in DELETE
$ curl --get -X DELETE https://google.com 
```

Reply:
Noted.